plugins/woocommerce-blocks/tests/e2e/test-1.spec.ts

```diff
@@ -0,0 +1,5 @@
+import { test, expect } from '@playwright/test';
```

Comment:
I think these are just dummy files that got pushed mistakenly. Are we using it anywhere?

Reply:
I've deleted `plugins/woocommerce-blocks/tests/e2e/test-1.spec.ts` and `plugins/woocommerce-blocks/tests/e2e/test-2.spec.ts`, which indeed looked like dummy files.